src/tracer/MOM_tracer_flow_control.F90

```diff
@@ -415,6 +415,17 @@ subroutine get_chl_from_model(Chl_array, G, GV, CS)
 
 end subroutine get_chl_from_model
 
+!> Returns pointers or values of members within the tracer_flow_control_CS type. For extensibility,
+!! each returned argument is an optional argument
+subroutine extract_tracer_flow_member(CS, use_MARBL_tracers)
+  type(tracer_flow_control_CS), target, intent(in)  :: CS !< module control structure
+  ! All output arguments are optional
+  logical,                    optional, intent(out) :: use_MARBL_tracers  !< If true, MARBL tracers are active
+
+  ! Constants within diabatic_CS
```

Comment:
This comment appears to have been cut and pasted from `extract_diabatic_member`.
It needs to be modified to match this context.